package.json

```diff
@@ -60,7 +60,7 @@
     "remark-parse": "^9.0.0",
     "remark-rehype": "^8.1.0",
     "seedrandom": "^3.0.5",
-    "sharetribe-flex-sdk": "^v1.20.1",
+    "sharetribe-flex-sdk": "https://github.com/sharetribe/flex-sdk-js.git#958956c269fe4ab6e44f616d31c444c2e5c1709c",
```

Comment:
TODO

Reply:
This is now changed. I noticed that the old version number contained there `v`, which seems wrong, so I didn't include it to the new one.